syntax = "proto3";
package hygge;
option cc_generic_services = true;

message ResultCode {
    int32 errcode = 1;  // 错误码
    bytes errmsg = 2;   // 错误信息
}

message LoginRequest {
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse {
    ResultCode res = 1;
    bool success = 2;  // 成功返回1，失败返回0
}

service UserServiceRpc {
    rpc Login(LoginRequest) returns(LoginResponse);
}

